docker/build-tools/Dockerfile

```diff
@@ -96,16 +96,18 @@ FROM golang:1.12 as build_hugo_context
 ENV HUGO_VERSION=0.55.5
 RUN mkdir -p /out/go/bin
 ADD https://github.com/gohugoio/hugo/releases/download/v${HUGO_VERSION}/hugo_${HUGO_VERSION}_Linux-64bit.tar.gz /tmp
-RUN tar -xzvf /tmp/hugo_${HUGO_VERSION}_Linux-64bit.tar.gz -C /tmp
-RUN mv /tmp/hugo /out/go/bin
+RUN tar -xzvf /tmp/hugo_${HUGO_VERSION}_Linux-64bit.tar.gz --strip-components=1 -C /usr/local
```

Comment:
Does the build_hugo_context need to use `golang:1.12`? We are also using `ubuntu:xenial`? Would that be a better fit. The steps could also be included in other contexts, but understand the reasoning behind separating out similar artifacts.

Reply:
for hugo - we could use scratch - its jsut a build context.

Reply:
agreed ubuntu:xenial would be a better fit - change made.